src/handleUninstall.ts

```diff
@@ -5,15 +5,15 @@ import { uninstalling } from "./binary/requests";
 
 export default function handleUninstall() {
   try {
-    const extensionsPath = path.dirname(tabnineContext.extensionPath);
+    const extensionsPath = path.dirname(tabnineContext.extensionPath!);
     const uninstalledPath = path.join(extensionsPath, ".obsolete");
     const isFileExists = (curr: fs.Stats, prev: fs.Stats) => curr.size != 0;
     const isModified = (curr: fs.Stats, prev: fs.Stats) =>
       new Date(curr.mtimeMs) >= new Date(prev.atimeMs);
-    const isUpdating = (files) =>
-      files.filter((f) =>
-        f.toLowerCase().includes(tabnineContext.id.toLowerCase())
-      ).length != 1;
+    const isUpdating = (files: string[]) =>
+      files.some((f) =>
```

Comment:
this is a bug - in normal case it will have one file named by the `id` for example tabnine-1.0.0
in case of an update, it should get more than one files prefixed with the `id` for example tabnine-1.0.1

so  some will always return true